cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -54,6 +54,12 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
       query("SHOW DATABASES", assertDatabasesShown) {
         resultTable()
       }
+      note {
+        p(
+          """Note that the results of this command are filtered according to the `ACCESS` privileges the user has.
```

Comment:
If a user has `CREATE/DROP DATABASE` or `DATABASE MANAGEMENT` privileges, all databases will be visible no matter what database they have `ACCESS` to. Also, the system database will always be visible in the list.